libs/beacon/benchmark/entropy_gen_bench.cpp

```diff
@@ -187,6 +187,7 @@ void EntropyGen(benchmark::State &state)
       nodes[m]->beacon_setup.StartNewCabinet(
           cabinet, test_attempt * entropy_rounds, start_time, prev_entropy,
           dealer.GetDkgKeys(nodes[m]->muddle_certificate->identity().identifier()));
+      nodes[m]->beacon_service.MostRecentSeen(test_attempt * entropy_rounds + entropy_rounds - 1);
```

Comment:
I know everyone knows BODMAS, but could you bracket: (test_attempt*entropy_rounds)+...

Reply:
Ah, sure.